pkg/cli/admin/catalog/mirror.go

```diff
@@ -647,6 +690,15 @@ func WriteManifests(out io.Writer, source, dest imagesource.TypedImageReference,
 	}
 
 	if dest.Type != imagesource.DestinationFile {
+		idmss, err := generateIDMSs(out, source.Ref.Name, idmsScope, maxIDMSSize, mapping)
```

Comment:
Now we are writing not only ICSP manifest but also IDMS, what is the expected behavior on 4.12 and 4.14?
In 4.12, icsp will be automatically used and in 4.14, idms will be used?

Reply:
The plan is to land IDSM in 4.13, and starting from 4.14, the openshift components migrate to use IDMS. There's an epic to keep track of this.


For this PR for 4.14, we can write both ICSP and IDMS manifests to make them both usable since ICSP and IDMS will both be functional until at least 4.17.